lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -105,7 +113,7 @@ def test_global_scatter_wrap_no_transform():
     plt.scatter(x, y, c=data)
 
 
-@ImageTesting(['global_map'])
+@ImageTesting(['global_map'], tolerance=16 if _proj4_version < 4.9 else 0.1)
```

Comment:
FYI, this one changed so much because it's cropped rather tightly. (It's really the only one with visible differences.)